Makefile

```diff
@@ -99,27 +99,40 @@ release: \
 
 release-victoria-metrics: \
 	release-victoria-metrics-amd64 \
+	release-victoria-metrics-linux-amd64 \
+	release-victoria-metrics-darwin-amd64 \
 	release-victoria-metrics-arm \
```

Comment:
This target should be removed because it is overridden by `release-victoria-metrics-linux-arm`